### DISCUSSION TITLE
Lightbox gallery for Tailwind

### DISCUSSION THREAD

**Discussion by cyggarro** [mdbootstrap/TW-Elements#1529] [Share ideas & request features]
This component should allow for the display of images or videos in a modal window that overlays the webpage. 

The lightbox should include the ability to customize the size and position of the modal window, as well as the ability to add caption text and navigation arrows. 

Additionally, it should allow for easy integration with JavaScript to control the display of the lightbox and the content that is displayed within it.

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1557

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
We added this to Tailwind Elements! You can now use the feature:
https://tailwind-elements.com/docs/standard/components/lightbox/
